book/src/intro.md

```diff
@@ -168,37 +168,37 @@ What's missing here is an easy and native way to represent mixed-type entities.
 In modern programming languages, such entities are enabled with a
 [sum type or tagged union](https://en.wikipedia.org/wiki/Tagged_union).
 
-While the original conception of the relational data model anticipated 
+While the original conception of the relational data model anticipated
 "product types" &mdash; in fact, describing a relation's schema in terms of
 a product type &mdash; it unfortunately did not anticipate sum types.
 
 > _Codd's original paper on the relational model has a footnote that
 > essentially describes as a product type:_
 >
-> <center><img src="codd-footnote.png" alt="Codd's Footnote" title="Codd's Footnote" width="500"></center>
+> ![Codd's Footnote](codd-footnote.png "Codd's Footnote")
```

Comment:
This happens to result in a noticeable change in the formatting. Rendered at tip of `main` the relevant section looks like this:

<img width="771" height="255" alt="image" src="https://github.com/user-attachments/assets/47fe7119-5f6c-4d46-aac7-9ae1ed261fee" />

Whereas on this branch it looks like this:

<img width="771" height="301" alt="image" src="https://github.com/user-attachments/assets/9956b8b4-3361-4c2a-a40c-c0ee16b83cc8" />

I don't necessarily find the new rendering distasteful, but since @mccanne wrote the original text he might have used HTML with that specific `width` setting to get the formatting just how he wanted it, so checking if he wants to chime in here.